src/functions-reference/array_operations.Rmd

```diff
@@ -179,6 +179,24 @@ The squared Euclidean distance between x and y
 `real` **`squared_distance`**`(row_vector x, row_vector y)`<br>\newline
 The Euclidean distance between x and y
 
+### Quantile
+
+Produces sample quantiles corresponding to the given probabilities.
+The smallest observation corresponds to a probability of 0 and the largest
+to a probability of 1.
+
+<!-- real; quantile; (data real[] xs, data real p); -->
+\index{{\tt \bfseries quantile }!{\tt (data real[] x, data real p): real}|hyperpage}
+
+`real` **`quantile`**`(data real[] xs, data real p)`<br>\newline
```

Comment:
We ended up supporting:

```
real quantile(data int[] xs, data real p)
```

as well. I guess just add integer signatures as well.

In some places we've written things like:

```
real quantile(data T[] xs, data real p)
```

where T can be a variety of things. My guess here is that the extra signatures will be kinda verbose, but not so much we need to switch to `T[]`.

Reply:
> We ended up supporting:

both for array and matrix operations, right?

> I guess just add integer signatures as well.

You mean in stanc3?